activiti-spring-boot/spring-boot-starters/activiti-spring-boot-starter/src/test/java/org/activiti/starter/tests/cmdendpoint/CommandEndpointIT.java

```diff
@@ -0,0 +1,417 @@
+ *
+ */
+
+package org.activiti.starter.tests.cmdendpoint;
+
+import java.util.Collection;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.UUID;
+import java.util.concurrent.atomic.AtomicBoolean;
+
+import org.activiti.services.core.model.ProcessDefinition;
+import org.activiti.services.core.model.ProcessInstance;
+import org.activiti.services.core.model.Task;
+import org.activiti.services.core.model.commands.ActivateProcessInstanceCmd;
+import org.activiti.services.core.model.commands.ClaimTaskCmd;
+import org.activiti.services.core.model.commands.CompleteTaskCmd;
+import org.activiti.services.core.model.commands.ReleaseTaskCmd;
+import org.activiti.services.core.model.commands.StartProcessInstanceCmd;
+import org.activiti.services.core.model.commands.SuspendProcessInstanceCmd;
+import org.activiti.services.core.model.commands.results.ActivateProcessInstanceResults;
+import org.activiti.services.core.model.commands.results.ClaimTaskResults;
+import org.activiti.services.core.model.commands.results.CommandResults;
+import org.activiti.services.core.model.commands.results.CompleteTaskResults;
+import org.activiti.services.core.model.commands.results.ReleaseTaskResults;
+import org.activiti.services.core.model.commands.results.StartProcessInstanceResults;
+import org.activiti.services.core.model.commands.results.SuspendProcessInstanceResults;
+import org.activiti.starter.tests.keycloak.KeycloakEnabledBaseTestIT;
+import org.junit.Before;
+import org.junit.ClassRule;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.autoconfigure.EnableAutoConfiguration;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.boot.test.web.client.TestRestTemplate;
+import org.springframework.cloud.sleuth.Tracer;
+import org.springframework.cloud.stream.annotation.EnableBinding;
+import org.springframework.cloud.stream.annotation.StreamListener;
+import org.springframework.core.ParameterizedTypeReference;
+import org.springframework.hateoas.PagedResources;
+import org.springframework.http.HttpMethod;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.messaging.MessageChannel;
+import org.springframework.messaging.support.MessageBuilder;
+import org.springframework.test.annotation.DirtiesContext;
+import org.springframework.test.context.TestPropertySource;
+import org.springframework.test.context.junit4.SpringRunner;
+
+import static org.assertj.core.api.Assertions.*;
+
+@RunWith(SpringRunner.class)
+@SpringBootTest(webEnvironment = SpringBootTest.WebEnvironment.RANDOM_PORT)
+@TestPropertySource("classpath:application-test.properties")
+@DirtiesContext(classMode = DirtiesContext.ClassMode.AFTER_CLASS)
+@EnableBinding(MessageClientStream.class)
+public class CommandEndpointIT extends KeycloakEnabledBaseTestIT {
```

Comment:
Tests depending on keycloak should no longer extends KeycloakEnableBaseTestIT. The related header will be automatically added.